schemas/com.snowplowanalytics.snowplow.badrows/recovery_error/jsonschema/1-0-0

```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for errors reported by Snowplow Event Recovery",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow.badrows",
+    "name": "recovery_error",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "payload": {
+        "anyOf": [
```

Comment:
Alright, that's a very weird one. I'm against `$ref` because most tools won't recognise it, but at the same time there's no chance we're going to copy-paste those schemas here. Can I ask you to:

1. Try putting something invalid in `payload` and check if new Iglu Client actually invalidates it.
2. If it doesn't - can you switch to `{"enum": ["iglu:com.snowplowanalytics.snowplow.badrows...", ...]}`

Also, it feels that it should be `oneOf` as we want to match against exactly one of those schemas.